Automattic/wp-calypso
# Media: Remove validation store and related data component

#### Changes proposed in this Pull Request

* Media: Remove validation store and related data component

#### Testing instructions

* Verify that the removed components aren't used anywhere.
* Verify that tests pass.
* No further testing should be necessary.

Note: #39374, #39375 and #39377 precede this PR and should be merged first. Feel free to review this one separately, though.

Note: This PR is part of an iteration to remove the media validation store, see #39372 for the full effort that removes it completely.
